avocado/core/jobdata.py

```diff
@@ -140,9 +136,10 @@ def retrieve_references(resultsdir):
     Retrieves the job test references from the results directory.
     """
     recorded_references = _retrieve(resultsdir, TEST_REFERENCES_FILENAME)
+    # TODO: Remove if support for replaying jobs generated under older
+    # versions is also removed
```

Comment:
What I'm removing here is the file creation, so jobdata created under this version won't have the `urls` link to `test_references`.  One extra step, not covered here, would be to not *read* `urls` anymore, which would break compatibility with *replaying* older jobs.

I wasn't sure what the overall impression about this was, and given that we spend some effort in keeping compatibility (as in the hability to replay jobs) all the way to 36.x, I thought we'd remove the obvious part first.